tooling/scripts/test-categories.yaml

```diff
@@ -135,6 +135,7 @@ group-08:
   - crypto
   - crypto-pgp
   - jq
+  - langchain4j
```

Comment:
Probably better to move it into `group-02` since it's completing way before the others at the moment.

Reply:
Should be ok in fixup commit.